public/app/plugins/datasource/cloudwatch/datasourcecfg.cue

```diff
@@ -0,0 +1,82 @@
+// Copyright 2023 Grafana Labs
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package grafanaplugin
+
+import (
+	common "github.com/grafana/grafana/packages/grafana-schema/src/common"
+	"github.com/grafana/grafana/pkg/plugins/pfs"
+)
+
+// This file (with its sibling .cue files) implements pfs.GrafanaPlugin
+pfs.GrafanaPlugin
+
+composableKinds: DataSourceCfg: {
+	maturity: "merged"
+
+	lineage: {
+		seqs: [
+			{
+				schemas: [
+					{
+						#LogGroup: {
```

Comment:
Question: `#LogGroup` is something we define here: https://github.com/grafana/grafana/blob/1a347bfb7bc2dbe00e87f5130b057c8bebf9c676/public/app/plugins/datasource/cloudwatch/dataquery.cue#L172-L181

Can it be reused here instead of redefining it?